### PR TITLE
Add type definitions for redux-infinite-scroll

### PR DESCRIPTION
**Pull request by silkyfray**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [ ] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

If adding a new definition:
- [x] The package does not provide its own types, and you can not add them.
- [x] If this is for an NPM package, match the name. If not, do not conflict with the name of an NPM package.
- [ ] Create it with `dts-gen --dt`, not by basing it on an existing project.
- [ ] `tslint.json` should be present, and `tsconfig.json` should have `noImplicitAny`, `noImplicitThis`, and `strictNullChecks` set to `true`.

### CODE REVIEW

File: types/redux-infinite-scroll/README.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# Installation
```
Comment (andy-ms) [inline types/redux-infinite-scroll/README.md]: We will generate this automatically for you. Don't include this on DefinitelyTyped.

File: types/redux-infinite-scroll/tsconfig.json [hunk 2]
```diff
@@ -0,0 +1,25 @@
+{
+    "compilerOptions": {
+        "module": "commonjs",
+        "lib": [
+            "es6",
+            "dom"
+        ],
+        "experimentalDecorators": true,
```
Comment (andy-ms) [inline types/redux-infinite-scroll/tsconfig.json]: Why enable this?

**Review by ghost — COMMENTED** [review]
Needs a `types/redux-infinite-scroll/redux-infinite-scroll-tests.ts` file containing tests.
Needs a `types/redux-infinite-scroll/tslint.json` file containing `{ "extends": "dtslint/dt.json" }`.

File: types/redux-infinite-scroll/tsconfig.json (hunk 2 above)
Comment (silkyfray) [inline types/redux-infinite-scroll/tsconfig.json]: Thank you for the feedback Andy! This is my first time submitting to DefinitelyTyped so I used other libraries to guide me. I will look closely on each one of your points.

**Pull request merged**